File: lab1/cs/ComputingThreadStatus.cs
public class ComputingThreadStatus {
	private ComputingThread _thread;
	private bool _canBreak;

public ComputingThread thread {
  get {
    return _thread;
  }
}
  public bool canBreak {
    get {
      return _canBreak;
    }
    set {
      _canBreak = value;
    }
  }
	
	public ComputingThreadStatus (ComputingThread thread) {
		this._thread = thread;
		this._canBreak = false;
	}
}

File: lab1/java/ComputingThreadStatus.java
public class ComputingThreadStatus {
	private final ComputingThread thread;
	private boolean canBreak;
	
	public ComputingThreadStatus (ComputingThread thread) {
		this.thread = thread;
		canBreak = false;
	}
	
	public ComputingThread getThread () {
		return thread;
	}
	
	
	public boolean isBreak () {
		return canBreak;
	}
	
	public void setBreak (boolean breakStatus) {
		canBreak = breakStatus;
	}
}
